src/components/shared/Header.js

```diff
@@ -34,6 +34,7 @@ const Header = (props) => {
         {props.isBack && (
           <ArrowBackIosIcon
             onClick={() => history.push("/venue")}
+            onClick={props.NavIcon}
```

Comment:
ada 2?
pake salah satu aja ya, thank you

Reply:
Compiled with warnings.

src/components/shared/Header.js
  Line 37:13:  No duplicate props allowed  react/jsx-no-duplicate-props

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.